<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>orgaEditWidget</class>
 <widget class="QWidget" name="orgaEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_25">
     <item>
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="0" column="0">
        <widget class="QLabel" name="photoLabel_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 10px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../images.qrc">:/images/ressources/anon.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="0" column="0" colspan="2">
          <widget class="QSplitter" name="splitter_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="prenomLineEdit_4">
            <property name="placeholderText">
             <string>Prénom</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="nomLineEdit_4">
            <property name="placeholderText">
             <string>Nom</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLineEdit" name="adresseLineEdit_4">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="placeholderText">
            <string>Adresse</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="emailLineEdit">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLineEdit" name="telephoneLineEdit_4">
             <property name="inputMask">
              <string>0000000000; </string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Téléphone</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="operateurLineEdit_4">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="placeholderText">
              <string>Opérateur téléphonique</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="8" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QLabel" name="confianceLabel_4">
             <property name="text">
              <string>Confiance</string>
             </property>
             <property name="buddy">
              <cstring>confianceComboBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="confianceComboBox_4">
             <item>
              <property name="text">
               <string>BdE</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Confiance</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Hard</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Soft</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="8" column="1">
          <widget class="QCheckBox" name="permisBcheckBox_4">
           <property name="text">
            <string>Permis B</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_22">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <widget class="QLabel" name="departementLabel_4">
               <property name="text">
                <string>Département</string>
               </property>
               <property name="buddy">
                <cstring>departementComboBox</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="departementComboBox_4">
               <item>
                <property name="text">
                 <string>PC</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GMC</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>IF</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GMD</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>TC</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GCU</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>BIM</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Bioch'</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GI</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GE</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>SGM</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GMPP</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GEN</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ext.</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_24">
             <item>
              <widget class="QLabel" name="anneeLabel_4">
               <property name="text">
                <string>Année</string>
               </property>
               <property name="buddy">
                <cstring>anneeSpinBox</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="anneeSpinBox_4">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="11" column="0">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QSplitter" name="splitter_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="surnomLineEdit_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>13</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string>Surnom</string>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget_6">
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QDateEdit" name="dateNaissanceEdit_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
               <property name="date">
                <date>
                 <year>1990</year>
                 <month>1</month>
                 <day>1</day>
                </date>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="10" column="0" colspan="2">
          <widget class="QTextEdit" name="notesTextEdit"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QDialogButtonBox" name="okCancelOrgas">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
